modules/sops/default.nix

```diff
@@ -144,17 +141,17 @@ in {
   };
   config = mkIf (cfg.secrets != {}) {
     assertions = [{
-      assertion = cfg.gnupgHome != null -> cfg.sshKeyPaths == [];
-      message = "Configuration options sops.gnupgHome and sops.sshKeyPaths cannot be set both at the same time";
+      assertion = (cfg.gnupgHome == null) != (cfg.sshKeyPaths == []);
+      message = "Exactly one of sops.gnupgHome and sops.sshKeyPaths must be set";
+    }] ++ concatLists (mapAttrsToList (name: secret: [{
+      assertion = builtins.pathExists secret.sopsFile;
```

Comment:
This might break the use case when you build your system in a CI where the sopsfile are not uploaded to. I think it should be only checked if `validateSopsFiles` is set.

Reply:
I wrapped the two assertions in `optionals cfg.validateSopsFiles`.